Taskfile.yml

```
@@ -1,23 +1,47 @@
 version: "3"
 
 includes:
-  docs:
-    taskfile: "docs/tasks.yml"
+  docs: "docs/tasks.yml"
+  lint: "lint-tasks.yml"
+  utils: "tools/yscope-dev-utils/taskfiles/utils.yml"
 
 vars:
   # Paths
   BUILD_DIR: "{{.ROOT_DIR}}/build"
+  CLP_FFI_PY_BUILD_DIR: "{{.BUILD_DIR}}/clp_ffi_py"
```

Using `clp_ffi_py` might not be a good idea. In this project, `clp_ffi_py` usually refers to the actual Python package, including the Python source and the installed ffi shared library. How about:
```suggestion
  CLP_FFI_PY_CMAKE_BUILD_DIR: "{{.BUILD_DIR}}/cmake_build"
```